testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -662,32 +662,59 @@ QUnit.module('check aria-labelledby attribute', {
         this.$textEditor = $('#texteditor');
         this.textEditor = this.$textEditor
             .dxTextEditor({
-                label: 'some'
+                label: 'custom',
+                placeholder: 'custom',
             })
             .dxTextEditor('instance');
         this.$input = this.$textEditor.find(`.${INPUT_CLASS}`);
         this.$label = this.$textEditor.find(`.${LABEL_CLASS}`);
+        this.$placeholder = this.$textEditor.find(`.${PLACEHOLDER_CLASS}`);
     }
 }, () => {
-    QUnit.test('if label is defined', function(assert) {
+    QUnit.test('if label and placeholder are defined', function(assert) {
```

Comment:
What we expect from the test? It is not clear from the name test.
Fix the tests names, pls

Reply:
[c05d065c557670099eaf3d0b77467a0ac6ccd2e1].